Kudu.Core/Infrastructure/PathUtils/PathWindowsUtility.cs

```diff
@@ -102,14 +104,20 @@ internal override string ResolveNpmJsPath()
         internal override string ResolveMSBuild15Dir()
         {
             string programFiles = SystemEnvironment.GetFolderPath(SystemEnvironment.SpecialFolder.ProgramFilesX86);
-            string[] probPaths = new[]{
+            List<string>probPaths = new List<string>(){
                 Path.Combine(programFiles, "Microsoft Visual Studio", "2017", "Enterprise", "MSBuild", "15.0", "Bin"), // visual studio Enterprise
                 Path.Combine(programFiles, "Microsoft Visual Studio", "2017", "Professional", "MSBuild", "15.0", "Bin"), // visual studio Professional
                 Path.Combine(programFiles, "Microsoft Visual Studio", "2017", "Community", "MSBuild", "15.0", "Bin"), // visual studio Community
-                Path.Combine(programFiles, "Microsoft Visual Studio", "2017", "BuildTools", "MSBuild", "15.0", "Bin"), // msbuild tools
+                Path.Combine(programFiles, "Microsoft Visual Studio", "2017", "BuildTools", "MSBuild", "15.0", "Bin") // msbuild tools
                 // above is for public kudu, below is for azure
-                Path.Combine(programFiles, "MSBuild-15.3.409.57025", "MSBuild", "15.0", "Bin")
                 };
+            if (DeploymentSettingsExtension.GetUseMSBuild_15_9())
+            {
+                probPaths.Add(Path.Combine(programFiles, "MSBuild-15.9", "MSBuild", "15.0", "Bin"));
```

Comment:
Let be specific in case we need to ship a fix.   Do we have info on minor and release number?

Reply:
No info on release number yet, I'll added it once i get it.